src/site/content/en/fast/codelab-two-ways-to-prefetch/index.md

```diff
@@ -0,0 +1,201 @@
+
+## Measure performance
+
+First establish the baseline performance:
+
+{% Instruction 'remix', 'ol' %}
+{% Instruction 'preview', 'ol' %}
+{% Instruction 'devtools-network', 'ol' %}
+
+1. In the **Throttling** drop-down list, select **Fast 3G** to simulate a slow connection type.
+1. To load the product page, click **Buy now** in the sample app.
+
+The `product-details.html` page takes about 600 ms to load:
+
+<img class="w-screenshot" src="./network-panel-one.png" alt="Network panel showing load times for product-details.html">
+
+## Prefetch the product page with `<link rel="prefetch">`
+
+To improve navigation, insert a `prefetch` tag in the landing page to prefetch the `product-details.html` page:
+
+- Add the following `<link>` element to the head of the `views/index.html` file:
+
+```html/7
+<!doctype html>
+  <html>
+    <head>
+       <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+
+      <link rel="prefetch" href="/product-details.html" as="document">
+	  ...
+</head>
+```
+
+The `as` attribute is optional but recommended; it helps the browser set the right headers and determine whether the resource is already in the cache.  Example values for this attribute include: `document`, `script`, `style`, `font`, `image`, and [others](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#Attributes).
+
+To verify that prefetching is working:
+
+{% Instruction 'preview', 'ol' %}
+{% Instruction 'devtools-network', 'ol' %}
+
+1. In the **Throttling** drop-down list, select **Fast 3G** to simulate a slow connection type.
+1. Clear the Disable cache checkbox.
+{% Instruction 'reload-app', 'ol' %}
+
+Now when the landing page loads, the `product-details.html` page loads too, but at the lowest priority:
+
+<img class="w-screenshot" src="./network-panel-two.png" alt="Network panel showing product-details.html prefetched.">
+
+The page is kept in the [HTTP cache](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching) for five minutes, after which the normal [`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) rules for the document apply. In this case, `product-details.html` has  a `cache-control` header with a value of `public, max-age=0`, which means that the page is kept for a total of five minutes.
+
+### Reevaluate performance
+
+{% Instruction 'reload-app', 'ol' %}
+1. To load the product page, click **Buy now** in the sample app.
+
+Take a look at the **Network** panel. There are two differences compared to the initial network trace:
+
+- The **Size** column shows "prefetch cache", which means this resource was  retrieved from the browser's cache rather than the network.
```

Comment:
```suggestion
- The **Size** column shows "prefetch cache", which means this resource was retrieved from the browser's cache rather than the network.
```